test/cli.js

```diff
@@ -364,6 +364,14 @@ test('bails when config contains `"tap": true` and `"watch": true`', t => {
 	});
 });
 
+test('bails when config does not contain `--concurrency` and `-c` values', t => {
```

Comment:
CLI invocation with `--concurrency` and `-c` without a value. Presumably there are tests where a value *is* provided, but if not those should be added.